src/Graphics/SurfaceFormat.cs

```diff
@@ -108,5 +108,13 @@ public enum SurfaceFormat
 		/// Byte encoding is in sRGB colorspace, read in shader in linear colorspace.
 		/// </summary>
 		Dxt5SrgbEXT,
+		/// <summary>
+		/// BC7 block texture format
+		/// </summary>
+		BC7EXT,
```

Comment:
We have to be goofy and do "Bc7EXT", emphasis on 'c', to stick with the XNA casing scheme. (Also I think Supergiant capitalizes it this way in their C# engine.)